config/config_test.go

```diff
@@ -263,6 +268,11 @@ func TestActiveConfig(t *testing.T) {
 					"min_length": 1024,
 					"compression_level": 1
 				}
+			},
+			"metrics": {
+				"http_requests": {
+					"buckets": [0.1, 0.2]
+				}
```

Comment:
This needs to be updated to adhere to the updated field format. Same for above.

Reply:
ah, missed that, thank you.
fixed, @ashutosh-narkar , please re-check